### DISCUSSION TITLE
Enchant Table → /is settings

### DISCUSSION THREAD

**Discussion by TheMartyCZ** [BG-Software-LLC/SuperiorSkyblock2#1443] [Denied Feature Requests] [answered]
I would like to request that an option be added to /is settings so that island owners can grant everyone access to enchant in the enchant table to other players.

**Comment by OmerBenGera** [comment, marked as answer]
Not gonna be added.
You can use an external plugin that opens an enchantment table with a command and make a button in the settings manager that will execute this command
